client/my-sites/email/form/mailboxes/components/mailbox-field/index.tsx

```diff
@@ -22,7 +23,7 @@ interface MailboxFormFieldProps {
 	isPasswordField?: boolean;
 	lowerCaseChangeValue?: boolean;
 	onFieldValueChanged?: ( field: MailboxFormFieldBase< string > ) => void;
-	onRequestFieldValidation: ( field: MailboxFormFieldBase< string > ) => void;
```

Comment:
`onRequestFieldValidation` receives `field` data from the handler that returns it.

`CreateCommonFieldPropsHandler` that generates a handler method that accepts a `field` as its first argument. That value is then used by `onRequestFieldValidation` internally. Passing `field` data directly to `onRequestFieldValidation` shouldn't be needed, as the method doesn't look for or use that argument any way.